frontend/play/index.html

```diff
@@ -12,8 +12,14 @@
     <link rel="stylesheet" href="css/play.css" type="text/css" />
     <link rel="stylesheet" href="css/fonts.css" type="text/css" />
     <script type="importmap"></script>
-    <script src="wasm_exec.js" defer></script>
-    <script src="index.js" type="module" defer></script>
+    <script>
+      const wasmImports = {
+        // Right is replaced for cache busting, e.g. "./module/evy.wasm": "./module/evy.11738cdd.wasm"
+        "./module/evy.wasm": "./module/evy.wasm",
+      }
+    </script>
+    <script src="wasm_exec.js" type="module"></script>
```

Comment:
I removed the `defer` here - apparently that's redundant with `type="module"`. if you want I'll move it into a separate commit.

Reply:
ok, I've moved this into a separate commit.